src/librustc_resolve/lib.rs

```diff
@@ -894,7 +895,7 @@ impl<'a> ModuleS<'a> {
 
     fn is_normal(&self) -> bool {
         match self.def {
-            Some(Def::Mod(_)) | Some(Def::ForeignMod(_)) => true,
```

Comment:
I have no idea what `is_normal` means and why `Def::ForeignMod` was removed from here :D 
Could you explain?